app/pages/order-task-list/contextCreator.js

```diff
@@ -1,12 +1,28 @@
 import neworderPageManifest from './neworder/manifest.json';
+import existingorderPageManifest from './existingorder/manifest.json';
 import taskListManifest from './taskListManifest.json';
 import { generateTaskList } from './helpers/generateTaskList';
 import { baseUrl } from '../../config';
 
-
-export const getContext = ({ orderId }) => ({
+export const getNewOrderContext = ({ orderId }) => ({
   ...neworderPageManifest,
   taskList: generateTaskList({ orderId, taskListManifest }),
   backLinkHref: `${baseUrl}/organisation`,
   orderId,
 });
+
+export const getExistingOrderContext = ({ orderId, orderDescription }) => ({
+  ...existingorderPageManifest,
+  title: `${existingorderPageManifest.title} ${orderId}`,
+  orderDescription,
+  taskList: generateTaskList({ orderId, taskListManifest }),
+  backLinkHref: `${baseUrl}/organisation`,
+  orderId,
+});
+
+export const getContext = ({ orderId, orderDescription }) => {
+  if (orderId === 'neworder') {
```

Comment:
a ternary might make this neater but not essential

Reply:
We had it like that, linting complained

Reply:
ok the ternary option looks like this. Think the if version is bit more readable
```
export const getContext = ({ orderId, orderDescription }) => (orderId === 'neworder'
  ? getNewOrderContext({ orderId })
  : getExistingOrderContext({ orderId, orderDescription }));
```

Reply:
although personally I like this better